src/service/git/git-api-service.ts

```diff
@@ -125,15 +125,18 @@ export class GitAPIService {
   ): Promise<string> {
     try {
       // check whether there is a fork with the same name as repo name
-      const repoName = await this.checkIfRepositoryExists(targetOwner, repo);
+      this.logger.info(`Checking if ${targetOwner}/${repo} is forked to ${sourceOwner}/${repo}`);
+      const repoName = await this.checkIfRepositoryExists(sourceOwner, repo);
 
       if (repoName) {
+        this.logger.info(`Found fork in ${sourceOwner}/${repo}`);
```

Comment:
to facilitate log reading (following same pattern like `Fork X whatever the result`
```suggestion
        this.logger.info(`Fork ${sourceOwner}/${repo} found.`);
```